INSTALL.md

```diff
@@ -14,6 +40,11 @@ have to build and install it from a fresh checkout of the GitHub tree.
     1.  The build scripts will use `cargo` from your path but you can set
         a different location by passing the `--cargo=/path/to/cargo` flag.
 
+    1.  The build scripts will use `go` from your path but you can set a
+        different location by passing the `--goroot=/path/to/goroot` flag.
+        You can also use the magic `none` value to disable Go usage, but
+        this will prevent running the integration tests or the code linter.
+
 1.  Run `make release` to download all required dependencies and build the
```

Comment:
Ran into:

```
error: failed to run custom build command for `fuse-sys v0.4.0-alpha (https://github.com/zargony/rust-fuse.git?rev=bbe450403abd47e719c922f3025edee727ea004b#bbe45040)`
process didn't exit successfully: `/Users/jingwen/code/sandboxfs/target/debug/build/fuse-sys-5f930ec2fd279142/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Command { command: "\"pkg-config\" \"--libs\" \"--cflags\" \"osxfuse >= 2.6.0\"", cause: Os { code: 2, kind: NotFound, message: "No such file or directory" } }', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

Had to `brew install pkg-config`.

Reply:
Added a note about this.